tox.ini

```diff
@@ -7,10 +6,11 @@ envlist=
   py35
 
 [testenv]
-deps=
-  -rrequirements.dev.txt
+passenv = TRAVIS TRAVIS_JOB_ID TRAVIS_BRANCH
+deps= -rtests_require
```

Comment:
We could simply add `tox` into setup.py's `tests_require`, so `python setup.py test` should work.